### PR TITLE
front end user profile

### PR DESCRIPTION
**Pull request by JackyLiu97**
Creating a user profile that includes, wishlist for user to wish to borrow, item list that show items the user have to sell/lend, and personal information like pic, age, school, major, name, etc.

### CODE REVIEW

**Comment by JackyLiu97** [comment]
Okay I will add a photo of themselves

File: client/src/js/components/App.js [hunk 1]
```diff
@@ -21,6 +23,7 @@ class App extends Component {
             <Route exact path="/register" component={Register} />
             <Route exact path="/login" component={Login} />
             <Route exact path="/products" component={ProductList} />
+            <Route exact path="/profile" component={Profile} />
```
Comment (yliu0715) [inline client/src/js/components/App.js]: What you are doing here is you created a static route for profile. But you can imagine that each user will render a different profile page based on their unique user content. So what you might want to look into is creating a dynamic route based on a unique user identifier.
Here is a link that might prove to be useful: https://learnwithparam.com/blog/dynamic-pages-in-react-router/

Comment (JackyLiu97) [reply client/src/js/components/App.js]: Thank you I will look into that

**Comment by JackyLiu97** [comment]
Okay I will write a unit test for it as well

File: client/src/js/components/App.js (hunk 1 above)
Comment (JackyLiu97) [reply client/src/js/components/App.js]: I'll set up the API call for User Profile in another PR. this is just to mainly set up the page.

**Pull request merged**